client/state/reader/follows/reducer.js

```diff
@@ -11,10 +11,83 @@ import {
 	READER_FOLLOW,
 	READER_UNFOLLOW,
 	READER_FOLLOWS_RECEIVE,
+	READER_SUBSCRIBE_TO_NEW_POST_EMAIL,
+	READER_SUBSCRIBE_TO_NEW_COMMENT_EMAIL,
+	READER_UPDATE_NEW_POST_EMAIL_SUBSCRIPTION,
+	READER_UNSUBSCRIBE_TO_NEW_POST_EMAIL,
+	READER_UNSUBSCRIBE_TO_NEW_COMMENT_EMAIL,
 } from 'state/action-types';
 import { prepareComparableUrl } from './utils';
 import { createReducer } from 'state/utils';
 
+function updatePostSubscription( state, { payload, type } ) {
+	const follow = find( state, { blog_ID: +payload.blogId } );
+	if ( ! follow ) {
+		return state;
+	}
+
+	const currentFollowState = get( follow, [ 'delivery_methods', 'email' ], {} );
+
+	const newProps = {
+		send_posts: ! ( type === READER_UNSUBSCRIBE_TO_NEW_POST_EMAIL )
```

Comment:
i don't think I can do that because the default state isn't present-but-false, it's missing-entirely.